stanford/bin/send-email.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from email.mime.text import MIMEText
+from subprocess import call
+import sys
+
+def send(recipient, sender, sender_name, subject, body):
+    with open('configuration/stanford/bin/email_params.txt', 'rt') as fin:
```

Comment:
It looks like this file isn't checked in?
I'm guessing that's because it has what looks to be an auth token inside?

Assuming that's the case, can we just template-ize that auth token,
commit the (no longer confidential) file here in the config repo, and then just
do a similar text replacement as we're already doing below?

`line = line.replace('AUTH_TOKEN', auth_token)`

Reply:
Oh, and to be explicit, we would then set that AUTH_TOKEN directly in the Jenkins job config and _not_ provide a real default/value in this file here.